app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,48 @@
+class AnswersController < ApplicationController
+  before_filter :require_user
+
+  def create
+    @node = DrupalNode.find(params[:nid])
+    @answer = Answer.new(nid: @node.id, uid: current_user.uid, content: params[:body])
```

Comment:
And here, it's more readable if we do one parameter per line, like:
```ruby
Answer.new({
  nid: @node.id,
  uid: current_user.uid,
  content: params[:body]
})
````
Can you update following this convention? Thanks!